src/main/java/com/shopify/model/ShopifyAddressUpdateRequest.java

```diff
@@ -0,0 +1,158 @@
+package com.shopify.model;
+
+import java.math.BigDecimal;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+@JsonInclude(Include.ALWAYS)
+public class ShopifyAddressUpdateRequest {
```

Comment:
This should contain a step builder like the rest of our request objects.

Reply:
@rjdavis3 this technically isn't the request that is getting sent. It's equivalent to the ShopifyAddress model, but wasn't sure what else to call it. The only difference is this model contains the fields always. Any ideas for a name here, and then I think we can omit the step builder here as well. I've added it to the actual request object ShopifyOrderAddressUpdateRequest

Reply:
@ryankazokas sorry yes I get that now. I think we only need one new step builder as part of this code review and that is within `ShopifyOrderShippingAddressUpdateRequest`. As long as `ShopifyOrderShippingAddressUpdateRequest` builds this `ShopifyAddressUpdateRequest` behind the scenes for our user then I am fine leaving this class as is.

```
ShopifyOrderShippingAddressUpdateRequest.newBuilder().withId(someShopifyOrderId).withAddress1(address1).withAddress2(address2)
...
.build();
```

I will resolve this conversation since I have a comment above on `ShopifySdk` about this.